tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql

```diff
@@ -16,6 +17,7 @@ from (
   inner join GmcDetails gmc on (gmc.id = cd.id)
    -- note: null values are filtered out by the condition below
     and lower(gmc.gmcNumber) <> 'unknown'
+    and gmc.gmcNumber not like CONCAT('%', UNHEX('c2a0'), '%') -- filter out all gmc number with non-breaking space
```

Comment:
I don't know why this is showing up as a new line, it's on the main branch

Reply:
Yesterday, I saw a message that this branch has conflicts with main branch. So might need to do a merge or rebase from main?